pkg/provider/yandex/certificatemanager/certificatemanager.go

```diff
@@ -38,8 +38,9 @@ func adaptInput(store esv1.GenericStore) (*common.SecretsClientInput, error) {
 	}
 	storeSpecYandexCertificateManager := storeSpec.Provider.YandexCertificateManager
 
-	if storeSpecYandexCertificateManager.Auth.AuthorizedKey.Name == "" {
-		return nil, errors.New("invalid Yandex Certificate Manager SecretStore resource: missing AuthorizedKey Name")
+	var authorizedKey *esmeta.SecretKeySelector
+	if storeSpecYandexCertificateManager.Auth.AuthorizedKey.Name != "" {
```

Comment:
Why isn't the name being empty not longer a problem? This will now, if the name is empty, pass in a nil to AuthorizedKey.

Reply:
Yes, it will be processed in buildSDK